doc/coadd1d.rst

```diff
@@ -284,16 +284,19 @@ be provided directly using the ``coadd1dfile`` parameter in the
 is ``coadd1d_<target>_<instrument name>_<YYYYMMDD>.fits`` or
 ``coadd1d_<target>_<instrument name>_<YYYYMMDD>-<YYYYMMDD>.fits``, if the coadd
 included more than one day's worth of data. The default location of the file
-will be along side the first spec1d file.  Currently ``instrument_name`` is
+will be along side the first spec1d file.  
+Currently ``instrument_name`` is
 taken from the ``camera`` attribute of the relevant
 :class:`~pypeit.spectrographs.spectrograph.Spectrograph` class.
 
-The format of the 1D coadd file follows follows the general
-class :class:`~pypeit.onespec.OneSpec`, such that its file extensions are:
+The format of the 1D coadd file follows the general
+class :class:`~pypeit.onespec.OneSpec`, such 
+that its file extensions are:
 
 .. include:: include/datamodel_onespec.rst
 
-You view the spectrum using the ``lt_xspec`` script, which loads the data
+You view the spectrum using the ``lt_xspec`` script 
+(``pypeit_show_1dspec`` will not work), which loads the data
```

Comment:
I've never used this... but is it worthwhile supporting this option? Maybe add this as an "enhancement" in a PypeIt issue to kick it down the road?

Reply:
I considered it but it would require one
too many assumptions about what the user 
was trying to do..